app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -6,6 +6,15 @@
   <div class="govuk-grid-column-two-thirds">
     <%= render ProviderInterface::StatusBoxComponent.new(application_choice: @application_choice, options: @status_box_options) unless @offer_present %>
 
+    <% if @application_choice.status == 'rejected' && @application_choice.reject_by_default_feedback_sent_at %>
+      <% feedback_at = @application_choice.reject_by_default_feedback_sent_at %>
+      <% intro = "You sent the following feedback to the candidate on #{feedback_at.strftime('%d %b %Y')}." %>
```

Comment:
Can we use the `to_s(:govuk_date)` here instead?

Reply:
In the previous version we needed `10 Nov 2020` rather than `10 November 2020` but we've switched to `to_s(:govuk_date)` now.